fpga/mqnic/ZCU106/fpga_zynqmp/ps/petalinux/project-spec/meta-user/recipes-bsp/fsbl-firmware/fsbl-firmware_%.bbappend

```diff
@@ -1,4 +1,4 @@
 ###############################################################################
 # enable message level FSBL_DEBUG_DETAILED
 
-YAML_COMPILER_FLAGS_append = " -DFSBL_DEBUG_DETAILED"
```

Comment:
So this is causing a build failure ... I saw your message in the Slack zynq channel.

The only explanation I do have currently is, that Xilinx changed the way to enable those detailed messages in FSBL in a newer versions. Or may be there is different, preferred way to enable them in newer versions. However up until now I did not see build failures due to that addition for recipe `fsbl-firmware`.

I always try to build the FSBL with detailed messages. If anything goes wrong within the FSBL, foremost due to board/SDcard issues/incompat, IMHO you won't get at least a partially decent bunch of messages without enabling `FSBL_DEBUG_DETAILED`.